akka-distributed-data/src/main/scala/akka/cluster/ddata/ORMultiMap.scala

```diff
@@ -3,22 +3,26 @@
  */
 package akka.cluster.ddata
 
-import akka.cluster.{ UniqueAddress, Cluster }
+import akka.cluster.{ Cluster, UniqueAddress }
 import akka.annotation.InternalApi
+import akka.cluster.ddata.ORMap._
 
 object ORMultiMap {
 
-  val _empty: ORMultiMap[Any, Any] = new ORMultiMap(ORMap.empty)
+  val _empty: ORMultiMap[Any, Any] = new ORMultiMap(ORMap.emptyWithORMultiMapTag, false)
+  val _emptyWithValueDeltas: ORMultiMap[Any, Any] = new ORMultiMap(ORMap.emptyWithORMultiMapTag, true)
   /**
    * Provides an empty multimap.
    */
   def empty[A, B]: ORMultiMap[A, B] = _empty.asInstanceOf[ORMultiMap[A, B]]
+  def emptyWithValueDeltas[A, B]: ORMultiMap[A, B] = _emptyWithValueDeltas.asInstanceOf[ORMultiMap[A, B]]
   def apply(): ORMultiMap[Any, Any] = _empty
 
   /**
    * Java API
    */
   def create[A, B](): ORMultiMap[A, B] = empty[A, B]
+  def createWithDeltaDelta[A, B](): ORMultiMap[A, B] = emptyWithValueDeltas[A, B]
```

Comment:
should be `createWithValueDeltas`, but I think `emptyWithValueDeltas` can be used for Java also, so it can be removed